src/glmtools.jl

```diff
@@ -165,19 +166,21 @@ Return a 3-tuple of the inverse link, the derivative of the inverse link, and wh
 The variance function is returned as NaN unless the range of μ is (0, 1)
 
 # Examples
-```jldoctest
-julia> inverselink(LogitLink(), 0.0)
-(0.5, 0.25, 0.25)
+```jldoctest; setup = :(using GLM)
```

Comment:
I wonder whether adding this in the manual wouldn't avoid the need to repeat this everywhere:
```julia

```@meta
CurrentModule = GLM
`` `

```

Reply:
Setting the current module that way doesn't seem to work for quick route of fixing outdated doctests:

```julia
julia> using Documenter, GLM
[ Info: Precompiling GLM [38e38edf-8417-5370-95a0-9cbb8c7f171a]

julia> doctest(GLM; fix=true)
```

But it's something we can revisit at some point.

Reply:
Too bad...